core/lib/slider.ts

```diff
@@ -389,16 +389,13 @@ export function createSlider(config?: PropsConfig<SliderProps>): SliderWidget {
 	});
 
 	// functions
-	const computeCleanValue = (value: number) => {
-		const min = min$(),
-			max = max$(),
-			stepSize = stepSize$();
+	const computeCleanValue = (value: number, min: number, max: number, stepSize: number) => {
```

Comment:
As this function no longer use anything from the closure, it could be moved out of it to avoid recreating it each time.